src/dex/pancakeswap-v3/config.ts

```diff
@@ -38,6 +38,21 @@ export const PancakeswapV3Config: DexConfigMap<DexParams> = {
       subgraphURL:
         'https://api.thegraph.com/subgraphs/name/pancakeswap/exchange-v3-bsc',
     },
+    [Network.ARBITRUM]: {
+      factory: '0x0BFbCF9fa4f9C56B0F40a671Ad40E0805A091865',
+      deployer: '0x41ff9AA7e16B8B1a8a8dc4f0eFacd93D02d071c9',
+      quoter: '0xB048Bbc1Ee6b733FFfCFb9e9CeF7375518e25997',
+      router: '0x1b81D678ffb9C0263b24A97847620C99d213eB14',
+      supportedFees: PANCAKE_SUPPORTED_FEES,
+      stateMulticall: '0xF8498aCeD3aFa417653415B8e32BAE9d764FBFf5',
+      uniswapMulticall: '0x1F98415757620B543A52E61c46B32eB19261F984',
```

Comment:
Are you sure this contract address is correct? I see it is named as `AlgebraInterfaceMulticall`, but for Mainnet in the same position different one: `UniswapInterfaceMulticall`. I don't know if it is just naming, but better to double check

Reply:
@Verisana https://arbiscan.io/address/0x1F98415757620B543A52E61c46B32eB19261F984#code vs https://etherscan.io/address/0xac1cE734566f390A94b00eb9bf561c2625BF44ea#code:

<img width="2397" alt="Computed Diff - Diff Checker 2023-08-10 12-34-10" src="https://github.com/paraswap/paraswap-dex-lib/assets/2093810/981997e0-85e5-48cf-b23c-c80086b70a23">


Also we use https://arbiscan.io/address/0x1F98415757620B543A52E61c46B32eB19261F984#code for UniV3 on Arbitrum https://github.com/paraswap/paraswap-dex-lib/blob/master/src/dex/uniswap-v3/config.ts#L73